doc/api/next_api_changes/behavior/23573-AL.rst

```diff
@@ -0,0 +1,5 @@
+All Axes have a ``get_subplotspec`` method now, which returns None for Axes not positioned via a gridspec
```

Comment:
Do you mean
```suggestion
All Axes have a ``get_gridspec`` method now, which returns None for Axes not positioned via a gridspec
```
?
AFAICS they have now both plus `set_subplotspec`. Can you please clarify this?

Reply:
Fixed (they all have both methods now).